src/features/public-api/server/apiAuth.ts

```diff
@@ -91,7 +91,9 @@ export async function verifyAuthHeaderAndReturnScope(
     console.error("Error verifying auth header: ", error);
     return {
       validKey: false,
-      error: error instanceof Error ? error.message : "Authorization error",
+      error:
+        (error instanceof Error ? error.message : "Authorization error") +
+        ". Confirm that you've configured the correct host.",
```

Comment:
Should we add a link to our docs here?

Reply:
we could, currently there is not the one page to refer to, will change with the docs relaunch, let's keep this in mind and add it then